sdk/nodejs/runtime/invoke.ts

```diff
@@ -224,6 +224,7 @@ async function invokeAsync(
     props: Inputs,
     opts: InvokeOutputOptions,
     packageRef?: Promise<string | undefined>,
+    checkDependencies?: boolean,
```

Comment:
Part of me is wondering do we go `checkDependencies?: "CheckDependencies" | "NoCheckDependencies"` here (or `check_dependencies` if we are snake-case for unions), but it's probably overkill/stupid.

Reply:
Feels overkill to me. Since this is an internal function, we can change this in the future if needed.